challenge-14/submissions/evassilyev/solution-template.go

```
@@ -0,0 +1,543 @@
+type UserServiceClient struct {
+	baseURL string
+}
+
+func NewUserServiceClient(conn *grpc.ClientConn) UserService {
+	// Extract address from connection for HTTP calls
+	// In a real gRPC implementation, this would use the connection directly
+	return &UserServiceClient{baseURL: "http://localhost:50051"}
+}
+
+func (c *UserServiceClient) GetUser(ctx context.Context, userID int64) (*User, error) {
+	resp, err := http.Get(fmt.Sprintf("%s/user/get?id=%d", c.baseURL, userID))
```

golangci-lint (2.13.2)

[error] net/http.Get must not be called. use net/http.NewRequestWithContext and (*net/http.Client).Do(*http.Request)

(noctx)